clients/cobol-lsp-vscode-extension/src/services/LanguageClientService.ts

```diff
@@ -171,6 +171,7 @@ export class LanguageClientService {
       args: [
         "-Dline.separator=\r\n",
         "-Ddialect.path=" + this.dialectsPath,
+        `-Duser.dir=${jarPath}`, // this makes sure that a java process started with UNC path like \\.\c:\Users doesn't fail
```

Comment:
java weird behavior
this makes sure that a java process started with UNC path like \\.\c:\Users doesn't fail